it/uk/gov/hmrc/gatekeeper/controllers/EmailsPreferencesControllerISpec.scala

```diff
@@ -374,6 +382,72 @@ class EmailsPreferencesControllerISpec extends ServerBaseISpec with BeforeAndAft
       }
     }
 
+    "GET /emails/email-preferences/all-users-new" should {
+      val selectedApis = Seq(combinedApi4, combinedApi5, combinedApi6)
+      val offset       = 0
+      val limit        = 15
+
+      "respond with 200 and render the page correctly on initial load when authorised" in {
+        primeAuthServiceSuccess()
+        primeFetchAllCombinedApisSuccess(combinedApis ++ selectedApis)
+        primeDeveloperServiceEmailPreferencesBySelectedTopicPaginated(allUsers, TopicOptionChoice.TECHNICAL, offset, limit)
+
+        val result = callGetEndpoint(s"$url/api-gatekeeper/emails/email-preferences/selected-user-topic?selectedTopic=TECHNICAL&offset=$offset&limit=$limit", validHeaders)
```

Comment:
Did you mean to have `all-users-new` in the url of this one, otherwise it's the same as the test after this one?

Reply:
Addressed